.env.sample

```diff
@@ -21,9 +21,6 @@ DJANGO_DEBUG=true
 # Debug level can conflict with Django Debug Toolbar
 DJANGO_LOGGING_LEVEL=INFO
 
-# department affiliation string from MCommunity settings
```

Comment:
Don't think this file diff is needed or is relevant to the issue.

Reply:
Yeah, looks like just a re-org, not related but a cleanup.

Reply:
It's not relevant, but it's something that needed to happen (either be commented out or removed). I can do it in another issue if you would like.

Reply:
Okay, @zqian, I've relented, see my latest commit undoing the change. I will create a separate issue about this (and maybe what @jonespm mentioned below).